libvmi/arch/arm_aarch32.c

```diff
@@ -78,11 +78,15 @@ void get_fine_second_level_descriptor(vmi_instance_t vmi, uint32_t vaddr, page_i
 // Based on ARM Reference Manual
 // Chapter B4 Virtual Memory System Architecture
 // B4.7 Hardware page table translation
-addr_t v2p_aarch32 (vmi_instance_t vmi,
+status_t v2p_aarch32 (vmi_instance_t vmi,
     addr_t dtb,
     addr_t vaddr,
     page_info_t *info)
 {
+    status_t status = VMI_FAILURE;
+
+    info->paddr = 0;
+    info->size = VMI_PS_UNKNOWN;
 
```

Comment:
page_info_t is already memset to zero by the callers so no need to reset it there